Workspaces/Inspector/Scripts/InspectorListViewController.cs

```diff
@@ -180,6 +180,15 @@ void UpdateItem(Transform t, float offset, bool dontSettle, ref bool doneSettlin
 			UpdateItemTransform(t, targetPosition, targetRotation, dontSettle, ref doneSettling);
 		}
 
+		protected override void RecycleItem(string template, InspectorListItem item)
+		{
+			var headerItem = item as InspectorHeaderItem;
+			if (headerItem)
+				headerItem.setLocked = null; // Reset, so it doesn't get called for the wrong object when re-used
```

Comment:
Only InspectorHeaderItems get setLocked toggled in GetItem.  Seems like this would still be needed here specifically for InspectorListItems.